githubscript/docker.d/worker.yml

```diff
@@ -42,26 +42,25 @@ github_projects:
         github_repo_name: staging-android-components
         contact_github: true
     'COT_PRODUCT == "xpi" && ENV == "prod"':
-      xpi-manifest:
+      mozilla-extensions/xpi-manifest:
```

Comment:
This will probably be `mozilla-extensions/*` when we roll out to production, so we can push releases to the various addons repos. (We don't need to worry about this while you're testing.)